src/sentry/integrations/github/webhook.py

```diff
@@ -71,10 +72,30 @@ def __call__(self, event):
 
 class InstallationEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#installationevent
-    def _handle(self, event, organization, repo):
+    def __call__(self, event):
+        installation = event['installation']
+        if installation and event['action'] == 'deleted':
+            integration = Integration.objects.get(
+                external_id=installation['id'],
+                provider='github',
+            )
+
+            self._handle(event, integration)
+
+    def _handle(self, event, integration):
         # TODO(maxbittker) these might need different behavior for missing
         # repository models in __call__
-        pass
+        orgs = {
+            org.id: org
+            for org in integration.organizations.all()
+        }
+        integration.update(status=ObjectStatus.DISABLED)
+
+        Repository.objects.filter(
+            organization_id__in=orgs.keys(),
```

Comment:
you can just do `organization_id__in=organizations.values_list('id', flat=True)` and remove the `orgs` variable since you don't actually use the org objects